content/post/reporting-is-a-gateway-drug.md

```diff
@@ -0,0 +1,70 @@
+
+
+## Steps to success
+So how do you take reporting lemons and make analytics lemonade?
+1. Same report, but faster
+2. Evolve the report
+3. Add insights
+
+
+### 1. Same report, but faster
+You may want to come in, analytics guns blazing and transform everything day one. Let me tell you why this is a terrible idea - your stakeholders don't know you (and you probably don't know them). This is how it will go:
+
+You: Hi, I am in Analytics. I just did XYZ analysis and you should now measure yourself against this new KPI I created
+Stakeholder: Um... no
+
+This is not stubbornness or resistance to change. Your stakeholders operate their part of the business based on a set of metrics that have evolved over time. These metrics, though there may be room for refinement, are familiar and understood. Even in a trusting Analytics-stakeholder relationship, introducing new KPIs (or refining current KPIs) is a challenging process. If there is no existing relationship, your odds of success are pretty much zero. I would also guess that you don't know nearly as much as you think you do about your stakeholders' area of the business and their challenges. There is probably a good reason they use these particular metrics.
+
+My suggestion is to start with the seemingly unambitious goal of producing the exact same (or a very similar) report, only faster. I can guarantee your business stakeholder will be supportive. Here is how the conversation will go:
+
+You: Hi, can I do your reporting for you?
+Stakeholder: I don't understand why you would want to do this, but I am definitely not going to say no
+
+At first it will not actually be faster. You are not just learning how to produce a single report, you are taking a crash course in your stakeholder's world. It is going to take a lot of work to understand the disparate data sources, the business logic and the subjective magic behind the report. However, once you understand how to create the report (and have felt the pain of actually owning the report) you will start to see opportunities to streamline the process. At first it will be little things like formulas or macros. Then maybe you hook up the Excel file directly to the database. Before you know it, the same report takes a fraction of the time. And what do you do with that extra time?
```

Comment:
Love this. If we had pull-quotes, I'd pull

>you are taking a crash course in your stakeholder's world.

You could consider elevating that sentence so it pops a bit more (either with italic formatting, or maybe finding a spot for it with more prominence).